components/expand-collapse/expand-collapse-content.js

```diff
@@ -35,6 +35,7 @@ class ExpandCollapseContent extends LitElement {
 	static get styles() {
 		return css`
 			:host {
+				--d2l-expand-collapse-content-transition-duration: 0.4s;
```

Comment:
I'm sure this is very subjective, but for me I only really notice the opacity change when collapsing... when expanding, by the time the content is tall enough to recognize it, it's pretty much fully visible. So maybe a slightly longer duration? 🤷

Reply:
Yeah, I think we're entering subjective area 😄 I'll push something up with a longer opacity duration and see how that looks

Reply:
Haha I can't tell the difference. But overall I agree that in the videos you provided it _feels_ a lot nicer with the fade. It feels a lot more "real" and less like we're just growing a container's height.

Reply:
ok -- so based on [this comment](https://github.com/BrightspaceUI/core/pull/3082#discussion_r1048536206) and through an offline convo with @dbatiste, I think we should keep the durations the same. It can introduce more complexity and race conditions from the `@transitionend`.

https://github.com/BrightspaceUI/core/blob/675b5939dc191df920bbdfc9ad44e6cf0ba565c3/components/expand-collapse/expand-collapse-content.js#L85

If we want to adjust the opacity further, I think tweaking the bezier curve is a simpler path forward.